### PR TITLE
change dash to bash.

### PR DESCRIPTION
**Pull request by crgaurav**
in this file, I have seen dash instead of bash.

### CODE REVIEW

File: develop/develop-images/dockerfile_best-practices.md [hunk 1]
```diff
@@ -463,7 +463,7 @@ RUN set -o pipefail && wget -O - https://some.site | wc -l > /number
 ```
 > Not all shells support the `-o pipefail` option.
 >
-> In such cases (such as the `dash` shell, which is the default shell on
+> In such cases (such as the `bash` shell, which is the default shell on
```
Comment (thaJeztah) [inline develop/develop-images/dockerfile_best-practices.md]: `dash` is actually correct; https://lwn.net/Articles/343924/

Comment (bermudezmt) [reply develop/develop-images/dockerfile_best-practices.md]: Oops, will revert.

Comment (thaJeztah) [reply develop/develop-images/dockerfile_best-practices.md]: Not sure it actually became the default though (so perhaps that part should be removed).

And, yes, the `dash` name is confusing (and not very commonly known)

Comment (thaJeztah) [reply develop/develop-images/dockerfile_best-practices.md]: The default (at least for the docker images) is "bash", but `dash` is definitely in there;

```bash
docker inspect --format '{{.Config.Cmd}}' debian:stretch
[bash]

docker run --rm debian:stretch sh -c 'which bash; which dash'
```

Comment (bermudezmt) [reply develop/develop-images/dockerfile_best-practices.md]: Can you please review https://github.com/docker/docker.github.io/pull/7770? Thanks!

**Pull request merged**